crates/net/discv4/src/lib.rs

```diff
@@ -537,11 +537,15 @@ impl Discv4Service {
         ) {
             InsertResult::Inserted => {
                 trace!( target : "net::disc",?record, "inserted new record to table");
-                self.notify(TableUpdate::Added(record));
+                self.notify(DiscoveryUpdate::Added(record));
             }
             InsertResult::ValueUpdated { .. } | InsertResult::Updated { .. } => {
                 trace!(target : "net::disc",?record,  "updated record");
             }
+            InsertResult::Failed(FailureReason::BucketFull) => {
```

Comment:
Does BucketFull + Discovered mean the following?

We already have 16 nodes in that bucket, so we're not gonna further discover via that peer, but we still want to connect to them